src/phases/tpch/denormalized/Q15.yml

```diff
@@ -4,57 +4,64 @@ Description: |
   Run TPC-H query 15 against the denormalized schema. Using an 'executionStats' explain causes each command to run its execution plan until no
   documents remain, which ensures that the query executes in its entirety.
 
-batchSize: &batchSize 101  # The default batch size.
+batchSize: &batchSize {^Parameter: {Name: "BatchSize", Default: 101}}
 query15Date: &query15Date {^Parameter: {Name: "Query15Date", Default: "1996-01-01"}}
 
+TPCHDenormalizedQuery15CreateView: &TPCHDenormalizedQuery15CreateView
+  create: &view revenue
```

Comment:
Done! Is `query15View` ok? I figured I don't really want to use revenue in the variable name since that's also a field in many queries and could get confusing. I updated both schemas.